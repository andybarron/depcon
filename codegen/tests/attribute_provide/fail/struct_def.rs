use depcon::*;

#[provide]
struct Bad;

fn main() {}
