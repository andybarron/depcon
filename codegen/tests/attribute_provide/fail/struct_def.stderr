error: expected `impl`
 --> tests/attribute_provide/fail/struct_def.rs:4:1
  |
4 | struct Bad;
  | ^^^^^^
